src/tests/whitespace_test

```diff
@@ -35,3 +35,15 @@ fi
             exit found
         }
     "
+
+declare found_file=0
+git ls-files | \
+    grep -v "^src/config/testconfigs/noparse.api.conf" | \
+    grep -v "^src/tests/cmocka/p11_nssdb/.*db" | \
+    while read file; do
+        test `tail -c 1 $file` && \
+            echo "no newline at eof: $file" && \
+            found_file=1
+    done
+
+[ $found_file -eq 1] && exit 1
```

Comment:
Another trick is to assign `true` or `false` to `found_file`. Then you can simply write this:

    $found_file && exit 1